webapp/packages/plugin-sql-editor-new/src/SQLEditor/SQLCodeEditorPanel/SQLCodeEditorPanel.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2023 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+import { observer } from 'mobx-react-lite';
+import { useState } from 'react';
+import styled, { css } from 'reshadow';
+
+import { useService } from '@cloudbeaver/core-di';
+import { NotificationService } from '@cloudbeaver/core-events';
+import { DATA_CONTEXT_NAV_NODE, getNodesFromContext, NavNodeManagerService } from '@cloudbeaver/core-navigation-tree';
+import { TabContainerPanelComponent, useDNDBox } from '@cloudbeaver/core-ui';
+import { classExtension, IEditorRef, LANG_EXT, ViewUpdate } from '@cloudbeaver/plugin-codemirror6';
```

Comment:
here we importing data from `@cloudbeaver/plugin-codemirror6` -> then we importing data from `codemirror6` synchronously. and then `SQLCodeEditorPanel` imported in `SQLCodeEditorPanelService` synchrounosly.
That lead to main application bundle increase
(you may fix it by movind logic and import from `@cloudbeaver/plugin-codemirror6` to `SQLCodeEditor` or using React.lazy in `SQLCodeEditorPanelService`)